emcc.py

```diff
@@ -918,6 +922,24 @@ def detect_fixed_language_mode(args):
       if separate_asm:
         shared.Settings.SEPARATE_ASM = os.path.basename(asm_target)
 
+      if 'EMCC_STRICT' in os.environ:
+        shared.Settings.STRICT = os.environ.get('EMCC_STRICT') != '0'
+
+      STRICT = ([None] + filter(lambda x: x.startswith('STRICT='), settings_changes))[-1]
```

Comment:
Sorry, I wasn't clear. My issue isn't with None vs False. It's that I don't understand that line - why does it need `[..something..]` at all? I guess it's so the `-1` can work, because we happen to just want the last one but the line just seems overcomplicated.

and we do this twice. how about adding a method, `get_last_change` which returns the last change, or None if there isn't one?